### PR TITLE
Login&Logout

### PR DESCRIPTION
**Pull request by noobsi**

### CODE REVIEW

**Comment by thaotonto** [comment]
ok

**Comment by noobsi** [comment]
signup

**Comment by noobsi** [comment]
login

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = t(".login_fail")
```
Comment (ducbet) [inline app/controllers/sessions_controller.rb]: ()

Comment (ducbet) [reply app/controllers/sessions_controller.rb]: flash.

**Comment by ducbet** [comment]
ok

**Pull request merged**